pkg/sql/logictest/testdata/logic_test/partial_index

```diff
@@ -455,3 +459,136 @@ SELECT * FROM d@f_b_s_foo_idx WHERE b AND s = 'foo'
 100  110  100  foo  true
 300  330  300  foo  true
 400  400  400  foo  true
+
+#### Backfill a partial index.
+
+statement ok
+CREATE TABLE e (a INT, b INT)
+
+statement ok
+INSERT INTO e VALUES (1, 10);
```

Comment:
just do 1 insert statement with all of the values?

Reply:
Done.